esp/scm/ws_esdlconfig.ecm

```diff
@@ -161,10 +161,12 @@ ESPrequest [nil_remove] ConfigureESDLBindingMethodRequest
 
 ESPresponse [exceptions_inline] ConfigureESDLBindingMethodResponse
 {
-    string EspProcName;         //Name of ESP Process
-    string EspBindingName;
-    string EsdlDefinitionID;    //The ESDL definition name.ver
-    string EsdlServiceName;     //which ESDL definition are we configuring
+    string EspProcName;                          //Name of ESP Process
+    [max_ver("1.1")] string ServiceName;         //which ESDL definition are we configuring
+    [max_ver("1.1")] int ServiceEsdlVersion;     //which version of the ESDL def are we configuring
+    [min_ver("1.2")] string EspBindingName;
+    [min_ver("1.2")] string EsdlDefinitionID;    //The ESDL definition name.ver
+    [min_ver("1.2")] string EsdlServiceName;     //which ESDL definition are we configuring
     ESPstruct BaseESDLStatus status;
```

Comment:
The what will be the behavior if a newer client populates a response from an older server without those fields?